flex-config/ui_attributes.common.json

```diff
@@ -263,7 +263,72 @@
           "location": false,
           "agent_skills": true
         }
+      },
+      "queues_view_data_tiles": {
+        "enabled": true,
```

Comment:
Given that this requires some configuration for the activity counts, maybe we shouldn't enable by default?

Reply:
done

Reply:
I think this change didn't make it in :) Could you please flip the `enabled` default to `false`?